ludwig/combiners/combiners.py

```diff
@@ -83,8 +83,16 @@ def __init__(
 
         if fc_layers is not None:
             logger.debug('  FCStack')
+
+            if self.flatten_inputs:
+                shapes = [np.prod(self.input_features[k].output_shape) for k in
+                          self.input_features]
+            else:
+                shapes = [self.input_features[k].output_shape[-1] for k in
+                          self.input_features]  # output shape not input shape
```

Comment:
Since this is on the common code path, can you double check that this doesn't break any other tests? E.g. running the commented out test cases in the `test_feature_multiple_outputs` function in `test_simple_features.py`?

E.g. test cases 5, 7, 8 (set features, vector input and output features) should pass

Reply:
For context, while trying to reproduce the `input_shape`/`output_shape` issue on my end, I'd commented out this snippet and used the original `first_layer_input_size`, and the bag features test case passed without an issue.

Reply:
Yeah, I've actually removed this change as part of discovering that `input_shape`/`output_shape` are actually working now. My theory is that the original error message was misleading somehow and was fixed by something else I fixed along the way.